static/logos.ts

```diff
@@ -0,0 +1,264 @@
+// Copyright (c) 2025, Compiler Explorer Authors
```

Comment:
This seems like a step back. Is there no better way to do this? We can serve these icons statically, surely?